### PR TITLE
Ensure uploaded assets have correct perms

### PR DESCRIPTION
**Pull request by aaannz**
see https://progress.opensuse.org/issues/17218

### CODE REVIEW

**Comment by aaannz** [comment]
Travis seems to have a really bad day today. 😿

File: t/full-stack.t [hunk 1]
```diff
@@ -224,6 +225,8 @@ wait_for_result_panel qr/Result: passed/, 'test 1 is passed';
 
 ok(-s "$resultdir/00000/00000001-$job_name/autoinst-log.txt",   'log file generated');
 ok(-s 't/full-stack.d/openqa/share/factory/hdd/core-hdd.qcow2', 'image of hdd uploaded');
+my $mode = S_IMODE((stat('t/full-stack.d/openqa/share/factory/hdd/core-hdd.qcow2'))[2]);
+is($mode, 420, 'exported image has correct permissions (420 -> 0644)');
```
Comment (coolo) [inline t/full-stack.t]: why not use octal in here? :)

Comment (aaannz) [inline t/full-stack.t]: Wish I knew what is going on in my head. 😕

Comment (coolo) [inline t/full-stack.t]: next time use 0x1A4 :)

**Pull request merged**